cmd/common.go

```diff
@@ -614,3 +631,116 @@ func exportCodesignFiles(tool Tool, archivePath, outputDirPath string) error {
 
 	return nil
 }
+
+func askUploadProvProfiles() (bool, error) {
+	messageToAsk := "Do you want to upload the provisioning profiles to Bitrise?"
+
+	answer, err := goinp.AskForBoolFromReader(messageToAsk, os.Stdin)
+	if err != nil {
+		return answer, err
+	}
+
+	return answer, nil
+}
+
+// ----------------------------------------------------------------
+// --- Upload methods
+
+// TODO
+func uploadProvisioningProfiles(profilesToUpload []profileutil.ProvisioningProfileInfoModel, outputDirPath string) error {
+	// Get accesToken from stdin
+	accessToken, err := uploaders.AskAccessToken()
+
+	appList, err := uploaders.FetchMyApps(accessToken)
```

Comment:
uploaders.FetchMyApps() is a strange function name. 

may the uploaders package should be: `bitriseclient`, this way:


```
client := bitriseclient.New(AccessToken)
client.FetchMyApps()
client.UploadProfiles() 
entc...
```